pkg/liqo-controller-manager/namespaceMap-controller/manage_remote_namespaces.go

```diff
@@ -12,29 +14,28 @@ import (
 	liqoconst "github.com/liqotech/liqo/pkg/consts"
 )
 
-// This function creates a new Namespace onto the remote cluster, the right client to use is chosen
-// by NamespaceMap's cluster-id.
+// This function creates a remote Namespace onto the remote cluster, if it doesn't exist yet.
+// The right client to use is chosen by means of NamespaceMap's cluster-id.
 func (r *NamespaceMapReconciler) createRemoteNamespace(clusterID, remoteName string) error {
 	if err := r.checkRemoteClientPresence(clusterID); err != nil {
 		return err
 	}
 
-	namespace := &corev1.Namespace{
+	remoteNamespace := &corev1.Namespace{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: remoteName,
 		},
 	}
 
-	if err := r.RemoteClients[clusterID].Create(context.TODO(), namespace); err != nil {
-		klog.Errorf("unable to create namespace '%s' onto remote cluster: '%s'", namespace.Name, clusterID)
```

Comment:
Now the error message is managed outside this function, where the situation is more clear... if the namespace already exists because I have previously created it or it was already present on that cluster

Reply:
Gotcha